.github/workflows/_bazel.yml

```diff
@@ -99,8 +99,13 @@ jobs:
           chmod +x llvm.sh
           sudo ./llvm.sh 19
           # keep in sync with build/ci.bazelrc
-          sudo apt-get install -y --no-install-recommends clang-19 lld-19 libunwind-19 libc++abi1-19 libc++1-19 libc++-19-dev libclang-rt-19-dev llvm-19
-          sed -i -e "s%llvm-symbolizer%/usr/lib/llvm-19/bin/llvm-symbolizer%" .bazelrc
+          sudo apt-get install -y -qq --no-install-recommends clang-19 lld-19 libunwind-19 libc++abi1-19 libc++1-19 libc++-19-dev libclang-rt-19-dev llvm-19
+          export LLVM_SYMBOLIZER_PATH=$(find /usr -path '*/llvm-19/bin/llvm-symbolizer' 2>/dev/null | head -1)
+          if [ -z "$LLVM_SYMBOLIZER_PATH" ]; then
+            echo "Warning: llvm-symbolizer-19 not found, make sure it is installed"
+            exit 1
+          fi
+          sudo ln -sf ${LLVM_SYMBOLIZER_PATH} /usr/bin/llvm-symbolizer
```

Comment:
How about just doing `sudo ln -s /usr/bin/llvm-symbolizer-19 /usr/bin/llvm-symbolizer` – I think that should work, the install paths of LLVM binaries are stable